src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenConditionalOperatorTests.cs

```diff
@@ -1061,9 +1062,9 @@ static void Main()
         bool b = true;
         System.Func<int> f = null;
         System.Console.WriteLine(f);
-        System.Func<int> g1 = b ? f : M;
+        System.Func<int> g1 = b ? f : new Func<int>(M);
```

Comment:
Retain the original IL, the case with IL that produces cache is moved to CodeGenMethodGroupConversionTests.cs